pkg/loop/config.go

```diff
@@ -11,13 +11,20 @@ import (
 )
 
 const (
-	envDatabaseURL            = "CL_DATABASE_URL"
-	envPromPort               = "CL_PROMETHEUS_PORT"
+	envDatabaseURL = "CL_DATABASE_URL"
+	envPromPort    = "CL_PROMETHEUS_PORT"
+
 	envTracingEnabled         = "CL_TRACING_ENABLED"
 	envTracingCollectorTarget = "CL_TRACING_COLLECTOR_TARGET"
 	envTracingSamplingRatio   = "CL_TRACING_SAMPLING_RATIO"
 	envTracingAttribute       = "CL_TRACING_ATTRIBUTE_"
 	envTracingTLSCertPath     = "CL_TRACING_TLS_CERT_PATH"
+
+	envTelemetryEndpoint         = "CL_TELEMETRY_ENDPOINT"
+	envTelemetryInsecureConn     = "CL_TELEMETRY_INSECURE_CONNECTION"
+	envTelemetryCACertFile       = "CL_TELEMETRY_CA_CERT_FILE"
```

Comment:
#nit naming unification
```suggestion
	envTelemetryTLSCertPath      = "CL_TELEMETRY_TLS_CERT_PATH"
```

Reply:
Why path instead of file? I was following the `beholder.Config` field names.

Reply:
Maybe we need to use the same naming for beholder config and existing tracing config because it will be part of the public config doc. But we can keep it as is